testsuite/tests/typing-objects/Exemples.ml

```diff
@@ -547,10 +547,7 @@ Error: Type
            "< cmp : int_comparable2 -> int; set_x : int -> unit; x : int >"
        is not a subtype of
          "int_comparable" = "< cmp : int_comparable -> int; x : int >"
-       Type "int_comparable" = "< cmp : int_comparable -> int; x : int >"
-       is not a subtype of
-         "int_comparable2" =
-           "< cmp : int_comparable2 -> int; set_x : int -> unit; x : int >"
+       Type "int_comparable" is not a subtype of "int_comparable2"
```

Comment:
Do you have an idea of why we lost the description of the types here ?

Reply:
Is the terrible code that uses `Marshal` to tell if something is big still there? That used to cause this kind of change whenever someone changed `types.ml`.

Reply:
The terrible code is still here, but only for module inclusion errors, and the check is disabled by default in expect tests.

Reply:
Here is a -1 lines fix: https://github.com/Octachron/ocaml/commit/3084345fb0ec98f5a35b1ae90331c1af4b6ebc76 .